### ISSUE TITLE
Error in packages on windows

### ISSUE THREAD

**Issue by leonardoXoliveira** [vtex/faststore#695]
![image](https://user-images.githubusercontent.com/28843438/119041304-d7ae0c80-b98c-11eb-851d-b147d1304c5f.png)

https://files.slack.com/files-pri/T01U12D64J3-F0237JDBAM6/image.png

**Comment by tlgimenes** [comment]
Thanks for contributing @leonardoXoliveira !

Can you share with us:
1. The steps to reproduce it
2. Environment you are using (a.k.a WSL, promp, etc)

Thanks!

**Comment by milaniromulo** [comment]
Hi @tlgimenes, i had the same problem here.
The steps were these: `git clone` > `yarn` to install > `yarn develop` to run.

This error appears in the terminal:
```
[gatsby-plugin-graphql] Error: Could not find fragment OrderFormFragment_orderForm for query SetRegionMutation
    at QueryManager.ensureRequiredFragments (D:\test\freewaystore.store\node_modules\@vtex\gatsby-plugin-graphql\manager.js:121:23)
    at D:\test\freewaystore.store\node_modules\@vtex\gatsby-plugin-graphql\manager.js:129:18
    at Array.map (<anonymous>)
    at QueryManager.getQueries (D:\test\freewaystore.store\node_modules\@vtex\gatsby-plugin-graphql\manager.js:128:31)
    at D:\test\freewaystore.store\node_modules\@vtex\gatsby-plugin-graphql\webpack.js:123:44
    at Hook.eval [as callAsync] (eval at create (D:\test\freewaystore.store\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:37:17)
    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (D:\test\freewaystore.store\node_modules\tapable\lib\Hook.js:18:14)
    at Watching._done (D:\test\freewaystore.store\node_modules\webpack\lib\Watching.js:199:28)
    at D:\test\freewaystore.store\node_modules\webpack\lib\Watching.js:127:21
    at Compiler.emitRecords (D:\test\freewaystore.store\node_modules\webpack\lib\Compiler.js:846:39)
    at D:\test\freewaystore.store\node_modules\webpack\lib\Watching.js:105:22
    at D:\test\freewaystore.store\node_modules\webpack\lib\Compiler.js:828:14
    at Hook.eval [as callAsync] (eval at create (D:\test\freewaystore.store\node_modules\tapable\lib\HookCodeFactory.js:33:10), <anonymous>:15:1)
    at Hook.CALL_ASYNC_DELEGATE [as _callAsync] (D:\test\freewaystore.store\node_modules\tapable\lib\Hook.js:18:14)
    at D:\test\freewaystore.store\node_modules\webpack\lib\Compiler.js:825:27
    at D:\test\freewaystore.store\node_modules\neo-async\async.js:2818:7
```

and in localhost, the same error appears in the @leonardoXoliveira print.

I'm using Windows 10 + PowerShell 7.0.3.
I tested in a macOS and just works with `yarn` if you install with `npm` there is also an error

**Comment by tlgimenes** [comment]
Thanks for contributing @milaniromulo!

Currently, we don't support `npm`, only `yarn`. Also, this seems like this babel/webpack plugin couldn't find the files to replace. I bet this is something to do with windows file system being non POSIX.

Can you do a test using [wsl](https://docs.microsoft.com/en-us/windows/wsl/install-win10)? This should be a POSIX compliant solution and this could work

**Comment by milaniromulo** [comment]
Hi @tlgimenes, sorry for the delay. We did tests with WSL and it's works :)
I think we can continue now, tks bro.

**Comment by leonardoXoliveira** [comment]
Hello @tlgimenes , thanks for clarification. 

At wsl, we face some issues with permission and stuff but we managed to solve. We will include this in the documentation if we have questions from other developers.

Tks!

**Comment by Gmantiqueira** [comment]
@tlgimenes I think it's better to fix the windows behavior instead of using WSL. Particularly, I don't have a good developing experience using WS, people reported that they had a slow developing experience using WSL and we have a good part of our external devs using Windows OS.

I investigated using Windows OS and we started having those problems after [this PR](https://github.com/vtex/faststore/pull/667). We are using a `process.cwd()` [there](https://github.com/vtex/faststore/pull/667/files#diff-cf9ccfafb1ed2311e879c56fb025cbdc18cf9f430675da6832c86ab1df826298R192) and I found that [this function is inconsistent across those platforms](https://github.com/nodejs/node/issues/8237), but I'm still reading about (it's from 2016, idk it's deprecated) and I'm comparing OSX and Windows behaviors.

I'm still investigating about this, so I'm reopening this issue.
